helios-services/src/main/java/com/spotify/helios/servicescommon/coordination/Paths.java

```diff
@@ -29,6 +29,7 @@
   private static final String JOBS = "jobs";
   private static final String JOBREFS = "jobrefs";
   private static final String HOSTS = "hosts";
+  private static final String REMOVED_HOSTS = "removed";
```

Comment:
I find it quite hard to understand what these node store, at a glance. Is it all removed hosts, as the name suggests?
Or is it actually `hosts-pending-undeploy-of-dg-job`?

Maybe document what is stored in them, or name it to reflect the contents better?